packages/core/src/components/QueryBuilder/index.ts

```diff
@@ -1 +1,7 @@
 export * from "./QueryBuilder";
+export {
+  defaultCombinators as hvQueryBuilderDefaultCombinators,
+  defaultLabels as hvQueryBuilderDefaultLabels,
+  defaultOperators as hvQueryBuilderDefaultOperators,
+} from "./Context";
+export * from "./types";
```

Comment:
We are currently making public all the types exported in `types` but we only want to export some of them (not `QueryAction`, `AskAction`, and `ValueComponentProps`). Thus, we should do something like: 

```suggestion
export type { HvQueryBuilderAttribute, HvQueryBuilderNumericRange, HvQueryBuilderDateTimeStrings, HvQueryBuilderDateTimeRange, HvQueryBuilderQueryRuleValue, HvQueryBuilderQuery, HvQueryBuilderQueryRule, HvQueryBuilderQueryCombinator, HvQueryBuilderQueryOperator, HvQueryBuilderLabels } from "./types";
```